Game/Source/GDKTestGyms/BenchmarkGymGameMode.cpp

```diff
@@ -89,6 +90,15 @@ void ABenchmarkGymGameMode::Tick(float DeltaSeconds)
 		const AActor* SpawnPoint = SpawnPoints[SpawnPointIndex];
 		SpawnNPC(SpawnPoint->GetActorLocation());
 	}
+	
+	if (HasAuthority() && SecondsTillPlayerCheck > 0.0f)
+	{
+		SecondsTillPlayerCheck -= DeltaSeconds;
+		if (SecondsTillPlayerCheck <= 0.0f && GetNumPlayers() != TotalPlayers)
```

Comment:
Logic reads a little strange - perhaps TotalPlayers should be TotalExpectedPlayers?